kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/menu/MenuUtilsTest.java

```diff
@@ -80,4 +84,17 @@ public void testBuildItemWithNormalIsWidget() {
         menuItem.setEnabled(false);
         assertFalse(menuItem.isEnabled());
     }
+
+    @Test
+    public void testBuildHasEnabledWidget() {
+        MenuUtils.HasEnabledIsWidget hasEnabledIsWidget = MenuUtils.buildHasEnabledWidget(button);
+
+        assertEquals(button, hasEnabledIsWidget.asWidget());
+        assertEquals(button.getTitle(), hasEnabledIsWidget.asWidget().getTitle());
+        assertEquals(false, hasEnabledIsWidget.isEnabled());
```

Comment:
`assertEquals(false, ..` -> `assertFalse`

Better yet use AssertJ consistently in the whole PR.